### ISSUE TITLE
Does rancher EBS storage plugin support tagging?

### ISSUE THREAD

**Issue by alexandrst88** [rancher/rancher#10007]
Does rancher EBS storage plugin support tags? For example I want to create ebs, set size and etc, but also is there possibility to set ebs tags?

**Comment by superseb** [comment]
Doesn't look like it's specified in the docs, but it is in the code: https://github.com/rancher/storage/blob/master/package/ebs/rancher-ebs#L157

If you have time to try it out and report back, I can label this as documentation and add it.

**Comment by alexandrst88** [comment]
one more question, is this possible to modify volume tags, name via Rancher CLI/UI?

**Comment by superseb** [comment]
No, these are created as volumes are requested in your service. If you want to change something, it needs to be done inside the volume definition. Even if you changed something, the definition in compose will be used when the container is moved so it wouldn't persist.

**Comment by superseb** [comment]
@alexandrst88 Were you able to test adding tags?
